ablateLibrary/eos/stiffenedGas.hpp

```diff
@@ -54,6 +54,7 @@ class StiffenedGas : public EOS {
     ComputeSpecificHeatFunction GetComputeSpecificHeatConstantVolumeFunction() override { return StiffenedGasComputeSpecificHeatConstantVolume; }
     void* GetComputeSpecificHeatConstantVolumeContext() override { return &parameters; }
     PetscReal GetSpecificHeatRatio() const { return parameters.gamma; }
+    PetscReal GetSpecificHeatCp() const { return parameters.Cp; }
```

Comment:
There is already a GetComputeSpecificHeatConstantVolumeFunction and GetComputeSpecificHeatConstantVolumeContext to get the specific heat.  We may want to just use these interface functions to help future proof any calls.